docs/src/less/base/index.less

```diff
@@ -200,6 +200,17 @@ main > div > h2 {
     margin: 8px;
 }
 
+.demo__signal {
+    align-items: flex-start;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+
+    > span {
+        margin: 5px;
```

Comment:
We should avoid adding margins in the demo code. All modules should themselves have sensible default for margin. The default is usually 0, 8px or 16px.